app/grants/clr.py

```diff
@@ -181,27 +182,60 @@ def get_totals_by_pair(contrib_dict):
 
         returns:
             pair totals between current round
-                {user_id (str): {user_id (str): pair_total (float)}}
-
+                {
+                    user_id (str): {
+                        user_id (str): pair_total (float)
+                    }
+                }
+            curr_agg_sqrts
+                {
+                    grantId (int): {
+                        user_id (float): sqrt(aggregated_amount) (float)
+                    }
+                }
+            pairs_tot
+                int: the number of cycles it took to construct the pairs
     '''
+    pairs_tot = 0
     pair_totals = {}
-
+    curr_agg_sqrts = {}
```

Comment:
Ah we keep updating this over time and before calculating -> check if data is available here.
this is nice :D